test/controllers/BankDetailsControllerSpec.scala

```diff
@@ -266,11 +276,12 @@ class BankDetailsControllerSpec extends ItSpec {
       status(result) shouldBe Status.SEE_OTHER
       redirectLocation(result) shouldBe Some(PageUrls.checkDirectDebitDetailsUrl)
       EssttpBackend.DirectDebitDetails.verifyUpdateDirectDebitDetailsRequest(TdAll.journeyId)
+      BarsStub.VerifyBusinessStub.ensureBarsVerifyBusinessNotCalled()
```

Comment:
here and in the tests below we could also verify details of the validate and verify personal call perhaps. We've got `ensureBarsVerifyPersonalCalled` for example but we don't call it here - the method also doesn't do anything beyond verifying the url. We could verify things like the request body as well to make it more cmoprehensive

Reply:
updated [here](https://github.com/hmrc/essttp-frontend/pull/99/commits/15b0d1a9a65da47d7c34805c84baed393a68b99e)